packages/analytics/analytics-utilities/src/types/explore-v4.ts

```diff
@@ -0,0 +1,136 @@
+// Query types
+import type { MetricUnit, RecordEvent } from './analytics-data'
+
+export const exploreFilterTypesV2 = ['in', 'not_in', 'selector'] as const
+export type ExploreFilterTypesV2 = typeof exploreFilterTypesV2[number]
+
+export const queryableExploreDimensions = [
+  'api_product',
+  'api_product_version',
+  'application',
+  'consumer',
+  'control_plane',
+  'control_plane_group',
+  'data_plane_node',
+  'gateway_service',
+  'route',
+  'status_code',
+  'status_code_grouped',
+  'time',
+] as const
+export type QueryableExploreDimensions = typeof queryableExploreDimensions[number]
```

Comment:
nit: spacing 
```suggestion

export type QueryableExploreDimensions = typeof queryableExploreDimensions[number]
```

Reply:
actually... the whole file could use some spacing between const/types 😬  but definitely just a nit

Reply:
Done.